migration/sql-files/005-work-item-links.sql

```diff
@@ -0,0 +1,76 @@
+-- Here's the layout I'm trying to create:
+-- (NOTE: work_items and work_item_types tables already exist)
+--        
+--           .----------------.
+--           | work_items     |        .-----------------.
+--           | ----------     |        | work_item_types |
+--     .------>id bigserial   |        | --------------- |
+--     |     | type text ------------>>| name text       |
+--     |     | [other fields] |    |   | [other fields]  |
+--     |     '----------------'    |   '-----------------'
+--     |                           |
+--     |    .-----------------.    |   .---------------------.
+--     |    | work_item_links |    |   | work_item_link_types|
+--     |    | --------------- |    |   | ---------           |
+--     |    | id uuid         | .------> id uuid             |
+--     .------source bigint   | |  |   | name text           |
+--      '-----target bigint   |/   |   | description text    |
+--          | type uuid ------'    ------source_type text    |
+--          | comment text    |     '----target_type text    |
+--          '-----------------'        | forward_name text   |
+--                                     | reverse_name text   |
+--    .--------------------------.    .- category uuid       |
+--    |work_item_link_categories |   / '---------------------'
```

Comment:
Do you think we should be adding some work_item_link_categories as common types relevant to the red hat hosted ALM ?